# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/arcload/hotfix-grub_error.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./arcgrub/xfs.c.vanilla	2024-12-05 13:30:29.793275123 +0100
+++ ./arcgrub/xfs.c	2024-12-05 13:30:49.103274978 +0100
@@ -193,7 +193,7 @@
     return grub_errno;
 
   if (grub_strncmp (inode->magic, "IN", 2))
-    return grub_error (GRUB_ERR_BAD_FS, "not a correct XFS inode.\n");
+    return grub_error (GRUB_ERR_BAD_FS, "not a correct XFS inode.");
 
   return 0;
 }
@@ -228,7 +228,7 @@
     }
 
   grub_error (GRUB_ERR_FILE_READ_ERROR,
-	      "xfs block %d for inode %d is not in an extent.\n",
+	      "xfs block %d for inode %d is not in an extent.",
 	      fileblock, grub_be_to_cpu64 (node->ino));
   return 0;
 }
